src/Features/Admin/Categories/AddCategory/index.js

```diff
@@ -53,20 +53,23 @@ const AddCategory = () => {
 
   const { mutate, mutateAsync, isLoading, error, isSuccess } = useMutation(category, {
     onSuccess: (response) => {
+      const resetFields = fields.map((field) => {
```

Comment:
we can have a initial state for fields and then just set that here so that we dont need to map over values